src/raftstore/store/peer.rs

```diff
@@ -1547,7 +1547,7 @@ impl Peer {
             pending_peers: self.collect_pending_peers(),
             written_bytes: self.peer_stat.last_written_bytes,
             written_keys: self.peer_stat.last_written_keys,
-            approximate_size: self.approximate_size().unwrap_or(0),
```

Comment:
Do we still need to keep approximate_size field?

Reply:
done